packages/flutter_tools/lib/src/commands/drive.dart

```diff
@@ -164,7 +159,58 @@ class DriveCommand extends RunCommandBase {
         );
       }
 
-      final LaunchResult result = await appStarter(this);
+      if (isWebPlatform && getBuildInfo().isDebug) {
+        // TODO(angjieli): remove this once running against
+        // target under test_driver in debug mode is supported
+        throwToolExit(
+          'Flutter Driver web does not support running in debug mode.\n'
+          '\n'
+          'Use --profile mode for testing application performance.\n'
+          'Use --release mode for testing correctness (with assertions).'
+        );
+      }
+
+      Uri webUri;
+
+      if (isWebPlatform) {
+        final FlutterProject flutterProject = FlutterProject.current();
+        final FlutterDevice flutterDevice = await FlutterDevice.create(
+          device,
+          flutterProject: flutterProject,
+          trackWidgetCreation: boolArg('track-widget-creation'),
+          target: targetFile,
+          buildMode: getBuildMode()
+        );
+        residentRunner = webRunnerFactory.createWebRunner(
+          flutterDevice,
+          target: targetFile,
+          flutterProject: flutterProject,
+          ipv6: ipv6,
+          debuggingOptions: DebuggingOptions.enabled(getBuildInfo()),
+          stayResident: false,
+          dartDefines: dartDefines,
+          urlTunneller: null,
+        );
+
+        // Sync completer so the completing agent attaching to the resident doesn't
```

Comment:
This comment isn't relevant to this code anymore

Reply:
Removed.

Reply:
Code is not updated yet..Seems like there are something wrong with Github.

Reply:
Code updated.